modules/solid_mechanics/include/materials/SolidModel.h

```diff
@@ -79,8 +79,10 @@ class SolidModel : public Material
   Function * const _poissons_ratio_function;
 
   const CRACKING_RELEASE _cracking_release;
-  const Real _cracking_stress;
+  Real _cracking_stress;
   const Real _cracking_residual_stress;
+  Function * const _cracking_stress_function;
```

Comment:
This may not be the "const" version you are expecting. If you place "const" after the asterisk it marks the pointer as constant (i.e. the pointer cannot be modified to point to something different). You may still change the object that you are pointing to. It's fine for this PR but just pointing that out, it's can be a source of confusion.